test/extended/authorization/per_node_validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "only-allow-name-matching-node-configmaps"
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+      - apiGroups:   [""]
+        apiVersions: ["v1"]
+        # notice you cannot restrict DELETEs.  Your per-node actor should not have deletion powers.
+        # it probably shouldn't have create powers either, but we can control that
+        operations:  ["CREATE", "UPDATE"]
+        resources:   ["configmaps"]
+  variables:
+    - name: hasNodeName
```

Comment:
I would expect such a policy to fail closed and consider a missing node name to be invalid.

Reply:
> I would expect such a policy to fail closed and consider a missing node name to be invalid.

Regular users and system:masters will not have node names, but should be allowed to update if they have the RBAC permissions.